lib/internal/webstreams/readablestream.js

```diff
@@ -1199,34 +1199,41 @@ ObjectDefineProperties(ReadableByteStreamController.prototype, {
   [SymbolToStringTag]: getNonWritablePropertyDescriptor(ReadableByteStreamController.name),
 });
 
+function TeeReadableStream(start, pull, cancel) {
```

Comment:
is there a reason this can't be a normal class? Would help with the inheritance/prototype hacks needed

Reply:
I'll have to call super and the original implementation don't call the constructor

Reply:
I think it would simplify the code significantly, not blocking or anything

Reply:
so you mean to just convert it to class with `extends` but without `super`?

when I do the wpt tests fails